OpenRA.Mods.Common/Activities/Turn.cs

```diff
@@ -11,18 +11,19 @@
 using System.Collections.Generic;
 using System.Linq;
 using OpenRA.Activities;
+using OpenRA.Mods.Common.Traits;
 using OpenRA.Traits;
 
 namespace OpenRA.Mods.Common.Activities
 {
 	public class Turn : Activity
 	{
-		readonly IDisableMove[] moveDisablers;
+		readonly Mobile mobile;
 		readonly int desiredFacing;
 
 		public Turn(Actor self, int desiredFacing)
 		{
-			moveDisablers = self.TraitsImplementing<IDisableMove>().ToArray();
+			mobile = self.TraitOrDefault<Mobile>();
```

Comment:
Can we please do this on `IMove` instead of reintroducing `Mobile` dependencies?